### ISSUE TITLE
Attempted symmetrical BNB/RUNE liquidity add, but only the RUNE fully went through

### ISSUE THREAD

**Issue by SirBenStokes** [asgardex/asgard-exchange#279]
Hello

I believe possibly related to issue [273](https://github.com/asgardex/asgard-exchange/issues/273) but slightly different in that the transaction partly went through it appears. 

I sent an add liquidity instruction with 7.884 RUNE / 0.2167 BNB (I know, not life changing sums) to the pool. However according to the transaction history it only added 0.0140 BNB, along with the full RUNE. As a result I assume it swapped the correct RUNE amount to match the lack of BNB as I now have a balanced amount of RUNE/BNB in my pool (screeshot) - but the rest of my BNB 0.2027 NB (0.2167 - 0.0140), which was instructed but didn't get added is not in my wallet. Having accounted for fees it still doesnt balance, unless I've missed something as I am relatively new to LP.

I started by transferring 0.0495 BNB from Binance and swapped about half to RUNE once in asgard. I did have some residual amount of BNB and RUNE held before this this whole process, from messing about earlier in the day.

Thorchain: thor1t5t5xg7muu3fl2lv6j9ck6hgy0970r08pvx0rz
BNB: bnb1txks3jy723rgk0uxe5559qfayug6zq4y8mypzg

Image: current pool stats

<img width="600" alt="image" src="https://user-images.githubusercontent.com/82573891/114839301-e88bc100-9dd5-11eb-8510-be76cdbb8484.png">

**Comment by SirBenStokes** [comment]
Quick addition - I suspect I have not gone through as a pool member, as I can not find my share in [viewblock](https://viewblock.io/thorchain/pool/BNB.BNB) 

Thank you.

**Comment by stuartkuentzel** [comment]
Hey @SirBenStokes, spent some time looking into this. Looking through the history on the BNB explorer, it looks like you tried twice to deposit to the BNB/RUNE pool.

https://explorer.binance.org/tx/A3C869BA64B8B3AB695445BC75026F0B8B1AF2FF9474CDFD2320A99790D8FE7E was picked up by THORChain as a successful pool
https://thorchain.net/#/txs/A3C869BA64B8B3AB695445BC75026F0B8B1AF2FF9474CDFD2320A99790D8FE7E

It looks like the symmetrical RUNE deposit failed for https://explorer.binance.org/tx/D8C7F80442C3191BBE64050A287238D62984DEAEBE721C5350B68442DE80C49D, which is an ongoing issue we're trying to resolve. You can see on https://thornode.thorchain.info/thorchain/pool/BNB.BNB/liquidity_providers, if you search your BNB address, it says "pending_asset" for the BNB amount, but "pending_rune" is 0.

We're working on an interface to deposit the failed RUNE side, once that is up, will drop you a link, you can resend the failed RUNE tx, and all your deposits will show up.

**Comment by SirBenStokes** [comment]
HI @stuartkuentzel .Ah yes it looks like the first of the links was from earlier in the day when I was testing with even smaller amounts.

Aaah regarding the second attempt you are saying it's the Rune which has actually failed. And it will not show up in my wallet either, and the upcoming interface will be required to be able to deposit it? 

Hugely appreciate your help, there must be a tonne of issues that must be incredibly time consuming so I really appreciate your time. I am sure there are people who are having trouble with much larger sums of money. At the end of the day I knew the risks so if there's trouble recovering my 75-100 USD worth it is not the total end of the world.

Thank you.

**Comment by stuartkuentzel** [comment]
@SirBenStokes yup, we have a pull request ready to recover stuck funds. Since THORChain is currently at pool cap, we'll have the option to withdraw the pending funds back into your wallet. We're waiting on a tiny Midgard API update to be rolled out next week. Once that's pushed on the THORChain end, we will push our code. Will go through all github issues related to this and drop the link for fund recovery, plus announce on telegram.

**Comment by stuartwk** [comment]
Hey @SirBenStokes, would you mind trying https://app.asgard.exchange/deposit-sym-recovery. Search for BNB with the "Search Pool" button. It should find the missing amount of RUNE you are missing. Since Pools are capped on the THORChain end, the only option at the moment is to "Withdraw All" (which will also withdraw anything successfully deposited in that specific pool). This should recover your pending BNB. Please keep in mind, you need at least 3 RUNE in your wallet to ensure enough gas fees are covered for the Withdraw. (it doesn't use 3 RUNE, but that's basically a safety net for THORChain transactions to go through).

Also, if there is an error, please do not try the transaction multiple times. Let me know if that works for you 👍

**Comment by SirBenStokes** [comment]
Hi @stuartwk. Ok I've given it a go, made sure I had over 3 RUNE too. Will let you know when it comes through. It successfully found the missing RUNE so just waiting for it to arrive back in my wallet. Although judging by the bellow message, after clicking withdraw, not sure if it is resubmitting or withdrawing, happy with either tbh

<img width="649" alt="Screenshot 2021-04-21 at 10 29 45" src="https://user-images.githubusercontent.com/82573891/115523447-5d0aa800-a28d-11eb-972b-a285271f1b6a.png">

**Comment by stuartwk** [comment]
@SirBenStokes thanks for bringing up the display message, that's a UI bug. It should withdraw..

**Comment by stuartwk** [comment]
Closing this out, as it's worked for other users. If it's still an issue, feel free to open a new ticket.
